cmd/kar-controllers/app/server.go

```diff
@@ -42,7 +43,7 @@ func buildConfig(master, kubeconfig string) (*rest.Config, error) {
 func Run(opt *options.ServerOption) error {
 	restConfig, err := buildConfig(opt.Master, opt.Kubeconfig)
 	if err != nil {
-		return err
+		return fmt.Errorf("[Run] unable to build server config, - error: %#v", err)
```

Comment:
In the special case of `fmt.Errorf()`, I think it's preferable to use the `%w` verb over `%v` to include errors.
https://pkg.go.dev/fmt#Errorf
https://stackoverflow.com/a/61287626

Reply:
Thanks @ronensc - I have updated those cases now.